demisto_sdk/__main__.py

```diff
@@ -2894,8 +2894,8 @@ def test_content(ctx, **kwargs):
     default=False,
 )
 @click.option(
-    "-pkw",
-    "--use-packs-known-words",
+    "-pkw/ -spkw",
+    "--use-packs-known-words/ --skip-packs-known-words",
```

Comment:
does it work with a space after the `/`?

Reply:
not sure but removed the space for any case..